mne/filter.py

```diff
@@ -1348,8 +1348,10 @@ def _mt_spectrum_proc(x, sfreq, line_freqs, notch_widths, mt_bandwidth,
     # max taper size chosen because it has an max error < 1e-3:
     # >>> np.max(np.diff(dpss_windows(953, 4, 100)[0]))
     # 0.00099972447657578449
-    # so we use 1000 because it's the first "nice" number bigger than 953:
-    dpss_n_times_max = 1000
+    # so we use 1000 because it's the first "nice" number bigger than 953.
+    # but if we have a new enough scipy,
+    # it's only ~2.5 sec for 8 tapers even with 1e7 samples (167 min @ 1000 Hz)
+    dpss_n_times_max = int(1e7) if check_version('scipy', '1.1') else 1000
```

Comment:
isn't this a bit an overkill? 2.5s is a lot if you do it hundreds of time in a loop.
1e6 maybe?

Reply:
In general you won't really loop over this -- usually you create the windows once then loop over channels and epochs. But yes I can decrease it

Reply:
1e6 is 1.6 sec, 1e5 is 0.17 sec so I'll go with 1e5.